pkg/provision/providers/qemu/controller.go

```diff
@@ -87,8 +87,12 @@ func (c *Controller) Reboot() error {
 	c.mu.Lock()
 
 	if c.state == PoweredOff {
+		c.state = PoweredOn
```

Comment:
this looks weird when we set after `PoweredOff`, maybe add a comment

Reply:
also we don't check if the `VMCommandStart` worked?

Reply:
See the PowerOn code above - I simply do what it does, if the machine is powered off. Because the behavior we want is for it to power on.